layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```

Comment:
Somethink like:

`<html{{ if .Site.Params.opengraph }} prefix="og: http://ogp.me/ns#"{{ end }}>`

should do, feel free to update the PR, or if it is too hard, i can do that :)